ovos_PHAL_plugin_oauth/__init__.py

```diff
@@ -253,7 +253,8 @@ def generate_qr(self, url, skill_id, app_id):
         return f"{temp_dir}/{skill_id}_{app_id}_oauth_qr_{img_id}.png"
 
     def build_plugin_service_url(self, oauth_url, skill_id, app_id, client_id=None, client_secret=None):
-        redirect_uri = f"http://{self.local_flask_host}:{self.port}/auth/callback/{skill_id}_{app_id}"
```

Comment:
Can `self.local_flask_host` be removed completely, or should it be updated here?